contracts/ism-multisig/Cargo.toml

```
@@ -32,6 +32,7 @@ thiserror = { workspace = true }
 cosmwasm-schema = { workspace = true }
 
 hpl-interface = { workspace = true }
+secp256k1 = "0.27.0"
```

```suggestion
secp256k1 = { workspace = true }
```
...and can you move this line to above? it'll be good to specify the other workspace packages